src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms ease 0s;
```

Comment:
Always add transition property. All is default value, and it's hard for the browser to control all properties